package.json

```diff
@@ -14,7 +14,7 @@
     "flow": "flow src",
     "fix-xcode": "react-native-schemes-manager all",
     "postinstall": "yarn fix-xcode",
-    "postversion": "react-native-version"
+    "postversion": "react-native-version --reset-build"
```

Comment:
Sets build number to one, not actually required with current setup but makes sense to do so.